tests/tools/test_doctest.py

```diff
@@ -77,3 +78,23 @@ def test_doctest_invalid_import(capsys: "CaptureFixture") -> None:
 
     _, err = capsys.readouterr()
     assert "ModuleNotFoundError: No module named 'thisdoesnotexist'" in err
+
+
+@pytest.mark.skipif(
+    sys.platform.startswith("win"), reason="Permission denied during CI"
+)
+def test_doctest_on_directory(capsys: "CaptureFixture") -> None:
+    """
+    Check that correct error is reported if notebook contains unimportable imports.
```

Comment:
you might want to fix the docstring for this test. Rest of the changes looks good.

Reply:
:rofl: oops, thanks!